platforms-whitepaper/v1alpha1/paper.md

```diff
@@ -285,13 +282,15 @@ To counter this, platform teams should include product managers from the start
 to share roadmaps, gather feedback and generally understand and represent the
 needs of platform users.
 
-Another challenge when adopting platforms is choosing the right capabilities and
-experiences to enable first. Capabilities that are frequently required and
-undifferentiated, like pipelines, databases and observability, may be a good
-place to start. Platform teams may also choose to focus first on a limited
-number of engaged and skillful app teams. Detailed feedback from such teams
-improves the first platform experiences; and people from those teams help
-champion and evangelize the platform to later adopters.
+However, Platform teams can easily be overloaded in their efforts to support many
```

Comment:
Rather than "however", could we use a transition word which _follows on_ rather than _contradicts_ the previous? That is, "however" seems to say this is an exception to the previous statements, but I think it's in fact adding to them.

Could be something like "Another challenge to keep in mind is ensuring your platform team itself isn't cognitively overloaded! Ways to reduce load on the platform team include..."

Reply:
That works, thanks.

Reply:
```suggestion
Additionally, Platform teams can easily be overloaded in their efforts to support many
```